api-generation/src/main/scala/ai/h2o/sparkling/api/generation/common/IgnoredParameters.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.h2o.sparkling.api.generation.common
+
+object IgnoredParameters {
+  val deprecated: Seq[String] = Seq(
+    "r2_stopping", // All
```

Comment:
nit: I assume these are deprecated without replacement?

Reply:
The replacement is `stopping_rounds`, `stopping_metric` and `stopping_tolerance`. We removed that manally in `3.30.0.1-1`. https://0xdata.atlassian.net/browse/SW-1819

Reply:
Thanks!